Domain/Sia.Shared/Extensions/Mediatr/MockEventTypeData.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Sia.Shared.Extensions.Mediatr
+{
+    public class MockEventTypeData
```

Comment:
Did this end up getting used anywhere?

Reply:
yes, in the Gateway.Requests.Playbook.EventTypes.GetEventType GetEventTypeShortCircuit 
method GenerateMockAsync